types/src/v1.ts

```diff
@@ -60,6 +60,16 @@ export const types: RegistryTypes = {
         Token: "H160"
       }
     },
+    TokenId: "U128",
```

Comment:
Should be the primitive u128 to match `nft::Config::TokenId`:
```suggestion
    TokenId: "u128",
```

Reply:
Good catch, fixed